extensions/authentication-jwt/src/__tests__/fixtures/controllers/user.controller.ts

```diff
@@ -95,4 +95,39 @@ export class UserController {
   async whoAmI(): Promise<string> {
     return this.user[securityId];
   }
+  @post('/signup', {
+    responses: {
+      '200': {
+        description: 'User',
+        content: {
+          'application/json': {
+            schema: {
+              'x-ts-type': User,
+            },
+          },
+        },
+      },
+    },
+  })
+  async signUp(
+    @requestBody({
+      content: {
+        'application/json': {
+          schema: getModelSchemaRef(NewUserRequest, {
+            title: 'NewUser',
+          }),
+        },
+      },
+    })
+    newUserRequest: NewUserRequest,
+  ): Promise<User> {
+    const password = await hash(newUserRequest.password, await genSalt());
+    const savedUser = await this.userRepository.create(
+      _.omit(newUserRequest, 'password'),
```

Comment:
failed to import lodash

Reply:
While we are here, try to remove the dependency on lodash.  `delete newUserRequest.password` might be suitable here or possible use destructing?

Reply:
> I think we have lodash in package.json so might we can use this. But @dougal83 your suggestion helped me some way out there. 

@madaky I believe that the general intent is to remove dependencies such as lodash over time where possible as they are not entirely necessary.